docs/platforms/flutter/index.mdx

```diff
@@ -71,6 +70,40 @@ Future<void> main() async {
 }
 ```
 
+```dart {tabTitle:FlutterWeb with WidgetsBindingIntegration}  {"onboardingOptions": {"performance": "13-15", "profiling": "16-19"}}
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+Future<void> main() async {
+  Sentry.runZonedGuarded(() async {
+    WidgetsBinding.ensureInitialized();
+
+    // Errors before init will not be handled by Sentry
+
+    await SentryFlutter.init(
```

Comment:
maybe explain why to use this method e.g in Flutter Web if you want to use WidgetsBindingIntegration before SentryFlutter.init

Reply:
Wrote a bit long winded explanation, maybe you can add your 2 cents to make it more concise.